src/current/v23.2/ui-hardware-dashboard.md

```diff
@@ -17,6 +17,20 @@ The **Hardware** dashboard displays the following time series graphs:
 
 {% include {{ page.version.version }}/ui/cpu-percent-graph.md %}
 
+## Host CPU Percent
+
+<img src="{{ 'images/v23.2/ui_host_cpu_percent.png' | relative_url }}" alt="DB Console CPU Percent graph" style="border:1px solid #eee;max-width:100%" />
+
+The **Host CPU Percent** graph displays the current user and system CPU percentage consumed by all processes on the host OS, normalized by number of cores, as tracked by the `sys.cpu.host.combined.percent-normalized` metric. If the CRDB process is run in a containerized environment, the host OS is the container since the CRDB process cannot inspect CPU usage beyond the container.
```

Comment:
```suggestion
The **Host CPU Percent** graph displays the current user and system CPU percentage consumed by all processes on the host OS, normalized by number of cores, as tracked by the `sys.cpu.host.combined.percent-normalized` metric. If the CockroachDB process is run in a containerized environment, the host OS is the container since the CockroachDB process cannot inspect CPU usage beyond the container.
```

Reply:
updated